shared/src/business/useCases/courtIssuedDocument/serveCourtIssuedDocumentInteractor.js

```diff
@@ -54,7 +57,7 @@ const completeWorkItem = async ({
 
   await applicationContext.getPersistenceGateway().putWorkItemInUsersOutbox({
     applicationContext,
-    section: user.section,
+    section: user.section ? user.section : DOCKET_SECTION,
```

Comment:
why would a user not have a section?  do you remember what scenario required this change?